tests/ApiTestCase.php

```diff
@@ -29,6 +29,7 @@ static function setUpBeforeClass(): void
       file_put_contents(self::TIER_TXT, KeymanHosts::TIER_TEST);
       self::$wroteTierTxt = true;
       KeymanHosts::Rebuild();
+      SetKeymanHostsForTest();
```

Comment:
Have to redo the hosts because Rebuild destroyed the overrides...

Reply:
Would it be safer to move SetKeymanHostsForTest() to the end of Rebuild?

Reply:
No, because Rebuild is a shared function, and SetKeymanHostsForTest is a site-local function. I know it's not very dry but this is probably the only place we will ever need it.